servers/services/src/main/java/org/projectnessie/services/cel/CELUtil.java

```diff
@@ -17,18 +17,22 @@
 
 import com.google.api.expr.v1alpha1.Decl;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import org.projectnessie.cel.checker.Decls;
 import org.projectnessie.cel.tools.ScriptHost;
+import org.projectnessie.cel.types.jackson.JacksonRegistry;
+import org.projectnessie.model.CommitMeta;
 
 /** A utility class for CEL declarations and other things. */
 public class CELUtil {
 
-  public static final ScriptHost SCRIPT_HOST = ScriptHost.newBuilder().build();
+  public static final ScriptHost SCRIPT_HOST =
+      ScriptHost.newBuilder().registry(JacksonRegistry.newRegistry()).build();
 
   public static final List<Decl> COMMIT_LOG_DECLARATIONS =
       Arrays.asList(
-          Decls.newVar("commit", Decls.newMapType(Decls.String, Decls.String)),
+          Decls.newVar("commit", Decls.newObjectType(CommitMeta.class.getName())),
           Decls.newVar("author", Decls.String),
```

Comment:
Do we still need to declare `author` + `committer` + ` commitTime` ?

Reply:
probably not. Updated the code and pushed a new version